examples/jsm/nodes/shadernode/ShaderNodeBaseElements.js

```diff
@@ -148,6 +150,7 @@ export const specularColor = nodeImmutable( PropertyNode, 'SpecularColor', 'colo
 
 export const reference = ( name, nodeOrType, object ) => nodeObject( new ReferenceNode( name, getConstNodeType( nodeOrType ), object ) );
 export const materialReference = ( name, nodeOrType, material ) => nodeObject( new MaterialReferenceNode( name, getConstNodeType( nodeOrType ), material ) );
+export const userData = ( name, inputType, userData ) => nodeObject( new UserDataNode( name, inputType, userData ) );
```

Comment:
I think it should be `getConstNodeType( inputType )`?

Reply:
For [this reason](https://github.com/mrdoob/three.js/pull/24053#issuecomment-1135523579) I didn't called `getConstNodeType()`. Maybe we remove it soon from the others `shader node elements`.